extensions/llama/python/llama.py

```diff
@@ -0,0 +1,68 @@
+from typing import Any
+from aiconfig import Output, Prompt
+from aiconfig.Config import AIConfigRuntime
+from aiconfig.default_parsers.parameterized_model_parser import ParameterizedModelParser
+from aiconfig.model_parser import InferenceOptions
+from aiconfig.schema import ExecuteResult
+
+from llama_cpp import CreateCompletionResponse, Llama
+
+from aiconfig.util.params import resolve_prompt
+
+
+SUPPORTED_MODELS = ["llama-2-7b-chat"]
+
+
+class LlamaModelParser(ParameterizedModelParser):
+    def __init__(self, model_path: str) -> None:
```

Comment:
Any reason to do file path vs folder to reuse a model parser for all local llama models?

Reply:
not exactly sure im following - you need to load a specific model right?

Reply:
Yes, but you could find the model based on name in the folder, so that we can register a single LlamaModelParser instance for any # of supported model names (associated with model binaries in the specified directroy) vs having one parser instance per model name (associated with model binary at the specified file path)